components/doner-card/index.js

```diff
@@ -0,0 +1,94 @@
+import { Grid, GridItem, Circle } from "@chakra-ui/layout";
+import { Box } from "@chakra-ui/react";
+import { PhoneIcon } from "@chakra-ui/icons";
+import { ImLocation } from "react-icons/im";
+
+// DonerCard Component can be resized by providing it with the width argument
+// other informations about the card can be provided to obtain reusability however we want.
+export default function DonerCard({
+  name = "Elyas Salar",
+  bloodType = "B+",
+  phoneNumber = "0772 106 8504",
+  city = "Chamchamal",
+  width = 300,
+}) {
+  let lWidth = width > 300 ? width.toString() + "px" : "300px";
+  let lHeight =
+    width > 300 ? (width * 0.4166666666666667).toString() + "px" : "125px";
```

Comment:
Those values are oddly specific, let's make the component responsive on 3 screen sizes by default.